spring-integration-ip/src/main/resources/org/springframework/integration/ip/config/spring-integration-ip-4.3.xsd

```diff
@@ -80,8 +80,17 @@
 							</xsd:appinfo>
 						</xsd:annotation>
 					</xsd:attribute>
-					<xsd:attribute name="host" type="xsd:string" />
+					<xsd:attribute name="host" type="xsd:string" use="optional"/>
 					<xsd:attribute name="acknowledge" type="xsd:string" />
+					<xsd:attribute name="socket-expression" type="xsd:string" use="optional">
+						<xsd:annotation>
+							<xsd:documentation>
+								An SpEL expression to be evaluated to determine which datagram
+								socket use for sending outgoing UDP packets (e.x. UDP inbound
```

Comment:
s/e.x./e.g.

Although English isn't my native language. So, I may just not know this case.